application/extractor/rootdomain.go

```diff
@@ -37,7 +37,7 @@ type Info struct {
 func InfoResponse(data []byte) (*Info, error) {
 	var infoMap interface{}
 	var contractErr *foundation.Error
-	_, err := insolar.UnMarshalResponse(data, []interface{}{&infoMap, &contractErr})
+	err := Generic(data, &infoMap, &contractErr)
```

Comment:
Generic doesn't describe what this procedure does.

Reply:
extractor.Generic() looks good to me. it can be used outside this package.  do you have better name?

Reply:
moved this into foundation package with documentation, example and tests